float64 threshold
---
bool resp